src/ruuvi.firmware.c.emProject

```diff
@@ -1059,9 +1305,9 @@
       <file file_name="../nRF5_SDK_15.3.0_59ac345/components/libraries/queue/nrf_queue.c" />
     </folder>
     <folder Name="System">
-      <file file_name="../nRF5_SDK_15.3.0_59ac345/modules/nrfx/mdk/ses_startup_nrf52811.s" />
+      <file file_name="../nRF5_SDK_15.3.0_59ac345/modules/nrfx/mdk/ses_startup_nrf52840.s" />
```

Comment:
Were these changes intentional? They seem to be breaking 52811 board

Reply:
Wrong board indeed. Fixed in latest commit to original parameters.